jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/LogEventHandler.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.maven;
+
+import com.google.cloud.tools.jib.event.events.LogEvent;
+import java.util.function.Consumer;
+import org.apache.maven.plugin.logging.Log;
+
+/** Handles {@link LogEvent}s by passing to the Maven {@link Log}. */
+class LogEventHandler implements Consumer<LogEvent> {
+
+  private final Log log;
+
+  LogEventHandler(Log log) {
+    this.log = log;
+  }
+
+  @Override
+  public void accept(LogEvent logEvent) {
+    switch (logEvent.getLevel()) {
+      case LIFECYCLE:
+        log.info(logEvent.getMessage());
+        break;
+
+      case DEBUG:
+        log.debug(logEvent.getMessage());
+        break;
+
+      case ERROR:
+        log.error(logEvent.getMessage());
+        break;
+
+      case INFO:
+        // Use lifecycle for progress-indicating messages.
```

Comment:
Should put this on `LogEvent.INFO` too.

Reply:
Also, lifecycle and info might be two different things in some situations (for example: gradle).

Reply:
oh oops, I didn't realize this was in the maven plugin. Brian's point still holds though. :)